server/socket/index.ts

```diff
@@ -1,147 +1,104 @@
-import WebSocket from 'ws';
+// Use https://admin.socket.io/#/ to monitor
+
+// TODO: turn on session affinity in all staging environments and in production to enable polling in transport options
+
+import { Server, Socket } from 'socket.io';
+import { instrument } from '@socket.io/admin-ui';
+import { BrokerConfig } from 'rascal';
 import * as jwt from 'jsonwebtoken';
+import { ExtendedError } from 'socket.io/dist/namespace';
 import http from 'http';
-import express from 'express';
-import * as net from 'net';
-import { WebsocketEventType, WebsocketMessageType, IWebsocketsPayload } from '../../shared/types';
-import { JWT_SECRET } from '../config';
-
+import { createAdapter } from '@socket.io/postgres-adapter';
+import { Pool } from 'pg';
+import { createCeNamespace, publishToCERoom } from './chainEventsNs';
+import { RabbitMQController } from '../util/rabbitmq/rabbitMQController';
+import RabbitMQConfig from '../util/rabbitmq/RabbitMQConfig'
+import { DATABASE_URI, JWT_SECRET } from '../config';
 import { factory, formatFilename } from '../../shared/logging';
 const log = factory.getLogger(formatFilename(__filename));
 
-const ALIVE_TIMEOUT = 30 * 1000; // heartbeats are 15 seconds
-const EXPIRATION_TIME = 15 * 60 * 1000; // 15 minutes, same as session expiration
 
-class AuthWebSocket extends WebSocket {
-  isAlive?: boolean;
-  aliveTimer?: NodeJS.Timeout;
-  expirationTimer?: NodeJS.Timeout;
-  isAuthenticated?: boolean;
-  user?: any;
+const origin = process.env.SERVER_URL || "http://localhost:8080"
+
+export const authenticate = (socket: Socket, next: (err?: ExtendedError) => void) => {
+	if (socket.handshake.query?.token) {
```

Comment:
Needs linting also -- hard tabs.

Reply:
Will do - not sure why this got so messed up - doesn't look like this locally.